polynote-frontend/webpack.config.js

```diff
@@ -50,7 +51,22 @@ module.exports = {
         { from: 'favicon.ico', to: 'favicon.ico' },
         { from: 'favicon.svg', to: 'favicon.svg' },
       ]
-    })
+    }),
+    {
+      apply: (compiler) => {
+        compiler.hooks.watchRun.tap('HtmlWebpackPlugin', (compilation) => {
+          if (compilation.modifiedFiles) {
+            let changedFiles = Array.from(compilation.modifiedFiles);
+            if (changedFiles.some(path => path.includes(".less"))) {
+              exec('./build_style', (err, stdout, stderr) => {
```

Comment:
Just wanna make sure I get this right - run the `./build_style` plugin, then if there is `stdout/stderr`, print that appropriately to the console? Will this give a message indicating the styles were successfully rebuilt (or failed)?